src/celeritas/io/ImportParameters.hh

```diff
@@ -43,13 +43,17 @@ struct ImportEmParameters
     double msc_lambda_limit{0.1};
     //! Kill secondaries below production cut
     bool apply_cuts{false};
+    //! Parameter used by the Coulomb scattering process for calculating
+    //! nuclear form factors (screening effect)
```

Comment:
everything is a parameter here, and doxygen may need this to be all on one line. Since the `WentzelOKandV` cross sections are also used by MSC let's note that as well.
```suggestion
    //! Nuclear screening factor for single/multiple Coulomb scattering
```